### PR TITLE
Load data with suspense

### PR DESCRIPTION
**Pull request by theosanderson**
Also bring map default parameters into the geojson

### CODE REVIEW

**Comment by theosanderson** [comment]
Will minify JSONs before merging

File: src/components/Covid19.jsx [hunk 1]
```diff
@@ -16,16 +16,17 @@ import LocationFilter from './LocationFilter'
...
-import { loadData } from '../utils/loadData'
 import useMobile from '../hooks/useMobile'
 import useLADs from '../hooks/useLADs'
 import useLineages from '../hooks/useLineages'
 import useLALookupTable from '../hooks/useLALookupTable'
 import useDates from '../hooks/useDates'
 
-const data = loadData()
+const Covid19 = ({ lineColor = 'blueGray', tiles, data }) => {
+  if (data.colors === undefined) {
```
Comment (richardgoater) [inline src/components/Covid19.jsx:26]: would suggest to move this to DataProvider and potentially put it in `useMemo` 👍🏽

**Review by richardgoater — APPROVED** [review]
Looks great dude, works really well and is very cutting edge :)

I left one comment about the default colours, and looks like there's some formatting issues around the `colors` props, but neither are critical.

File: src/components/Covid19.jsx (hunk 1 above)
Comment (theosanderson) [reply src/components/Covid19.jsx]: Great point - ended up putting this in the fetch logic that only gets called once

**Pull request merged**